File: 1_Arrays/Day_002_Move_All_Zeroes_To_End/Solution.java

class Solution {
    void pushZerosToEnd(int[] arr) {
        // code here
        int size = arr.length;
        int[] tempArray = new int[size];
        int j = 0;
        for(int i=0;i < size;i++){
            if(arr[i]>0){
                tempArray[j]=arr[i];
                j++;
            }
        }
        for(int i=0;i < size;i++){
            arr[i] = tempArray[i];
        }
    }
}

File: 1_Arrays/Day_002_Move_All_Zeroes_To_End/Solution.py
class Solution:
	def pushZerosToEnd(self,arr):
		# code here
		n = len(arr)
		tempArray = [0]*n
		j=0
		for i in range(n):
			if(arr[i]>0):
				tempArray[j]= arr[i]
				j+=1
		
		for i in range(n):
			arr[i] = tempArray[i]